<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPaw</class>
 <widget class="QWidget" name="FormPaw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RAS include output file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOutfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory address to pack to</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePackedAddress">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unpack Address</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Packed file directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOutDir">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkExomizer3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Using exomizer 3+</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="leOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
